lib/publiccloud/ec2.pm

```diff
@@ -126,9 +134,7 @@ sub upload_img {
     my $img_arch = get_var('PUBLIC_CLOUD_ARCH', 'x86_64');
     my $sec_group = get_var('PUBLIC_CLOUD_EC2_UPLOAD_SECGROUP');
     my $vpc_subnet = get_var('PUBLIC_CLOUD_EC2_UPLOAD_VPCSUBNET');
-    my @instance_main_type = split(/\./, get_var('PUBLIC_CLOUD_INSTANCE_TYPE'));
-    my $instance_size = get_var('PUBLIC_CLOUD_IMAGE_LOCATION') =~ /-SAP-/ ? 'large' : 'micro';
```

Comment:
isn't you changing SAP related logic ? Have you confirmed this with SAP team ? @ricardobranco777 FYI

Reply:
I added this because SAP upload failed with smaller instance. But now is the instance large by default, so should be OK.